helpers/integration.go

```diff
@@ -546,3 +547,10 @@ func AddFinalizerToScenario(adapterClient client.Client, logger IntegrationLogge
 
 	return nil
 }
+
+func IsObjectYoungerThanThreshold(obj metav1.Object, threshold time.Duration) bool {
```

Comment:
Let's add a simple unit test for this function just in case.

Reply:
Agree to cover the more code in UT.